src/fundus/scraping/pipeline.py

```diff
@@ -208,11 +212,18 @@ def crawl(
 
         event_loop = asyncio.new_event_loop()
```

Comment:
Good point. Actually, that should raise an error if there is already a running event loop. We can't check if another loop exists, only if one is running. If so, this will now raise an error pointing to use `crawl_async` instead.